polyglot/piranha/src/models/language.rs

```diff
@@ -68,43 +75,34 @@ impl PiranhaLanguage {
 
 impl Default for PiranhaLanguage {
   fn default() -> Self {
-    Self {
+    let rules: Rules = parse_toml(include_str!("../cleanup_rules/java/rules.toml"));
+    let edges: Edges = parse_toml(include_str!("../cleanup_rules/java/edges.toml"));
+    PiranhaLanguage {
       name: default_language(),
-      supported_language: SupportedLanguage::Java,
+      supported_language: SupportedLanguage::default(),
       language: tree_sitter_java::language(),
-      rules: None,
-      edges: None,
-      scopes: vec![],
-      comment_nodes: vec![],
+      rules: Some(rules),
+      edges: Some(edges),
+      scopes: parse_toml::<ScopeConfig>(include_str!("../cleanup_rules/java/scope_config.toml"))
+        .scopes()
+        .to_vec(),
+      comment_nodes: vec!["line_comment".to_string(), "block_comment".to_string()],
     }
   }
 }
 
 impl From<&str> for PiranhaLanguage {
   fn from(language: &str) -> Self {
     match language {
-      "java" => {
-        let rules: Rules = parse_toml(include_str!("../cleanup_rules/java/rules.toml"));
-        let edges: Edges = parse_toml(include_str!("../cleanup_rules/java/edges.toml"));
-        PiranhaLanguage {
-          name: language.to_string(),
-          supported_language: SupportedLanguage::Java,
-          language: tree_sitter_java::language(),
-          rules: Some(rules),
-          edges: Some(edges),
-          scopes: parse_toml::<ScopeConfig>(include_str!(
-            "../cleanup_rules/java/scope_config.toml"
-          ))
-          .scopes()
-          .to_vec(),
-          comment_nodes: vec!["line_comment".to_string(), "block_comment".to_string()],
-        }
-      }
+      "java" => PiranhaLanguage::default(),
```

Comment:
Um, this still seems less reconfigurable than having the actual instantiation for the Java configuration here, and then calling `PiranhaLanguage::from` from `PiranhaLanguage::default`, the whole point is that default should be easy to change (even if technical hardcoded) and not entangled with the specifics of configuring Java...